commons/dev-public-catalog/values-cronjob.yaml

```
@@ -0,0 +1,21 @@
+namespace: "dev-public-catalog"
+
+cronjob:
+  env:
+    LOG_LEVEL: "info"
+  image:
+    repositoryPrefix: "505630707203.dkr.ecr.eu-south-1.amazonaws.com"
+    imagePullPolicy: Always
+    tag: $IMAGE_TAG_PLACEHOLDER
+    digest: $IMAGE_DIGEST_PLACEHOLDER
+  resources:
+    requests:
+      cpu: "1000m"
+      memory: "2Gi"
+    limits:
+      cpu: "1000m"
+      memory: "2Gi"
+
+local:
+  interop_public_catalog_commons:
+    AWS_ACCOUNT_ID: "505630707203"
```

```suggestion
  awsAccountId: "505630707203"
```
We can flatten and use camelCase